### PR TITLE
Show deleted files finalize

### PR DESCRIPTION
**Pull request by kevinrobbins**

### CODE REVIEW

**Comment by kevinrobbins** [comment]
[Implement a toggle in S3Commander to show deleted files and folders.](https://trello.com/c/zzXEGXir/2503-implement-a-toggle-in-s3commander-to-show-deleted-files-and-folders)

File: src/bucket/BucketController.ts [hunk 1]
```diff
@@ -167,6 +183,32 @@ export class BucketController {
         // store folders and files in alphabetical order
         this.folders = contents.folders.sort(compareObjectNames);
         this.files = contents.files.sort(compareObjectNames);
+
+        this.backend.getDeletedContents(this.bucket, this.currentFolder)
+        .then((contents: IFolderContents) => {
```
Comment (CtrlC-Root) [inline src/bucket/BucketController.ts]: This formatting is not consistent with the rest of the project. The `.then` and `.catch` lines need to be indented one level past the top-level `this.backend` line.

File: src/file/file.html [hunk 2]
```diff
@@ -1,8 +1,8 @@
-<div class="s3c-file">
+<div class="s3c-file" ng-class="{'deleted': $ctrl.file.deleted}">
   <!-- file link -->
   <a href="{{ $ctrl.file.getDownloadLink() }}"
     download="{{ $ctrl.file.getPath().name() }}"
-    target="_blank">{{ $ctrl.file.getPath().name() }}</a>
+    target="_blank" >{{ $ctrl.file.getPath().name() }}</a>
```
Comment (CtrlC-Root) [inline src/file/file.html]: Unnecessary whitespace.

**Comment by CtrlC-Root** [comment]
Overall the code looks reasonable. I would have modified the `getContents` method or added a new one called `getVersionedContents` which could be used to retrieve all available versions of each file and folder. Maybe we'll come back and revisit this later before making the next major release.

File: src/index.css [hunk 3]
```diff
@@ -195,3 +195,7 @@ div.s3c-controls label {
   /* why is this necessary??? */
   padding: 3px 6px;
 }
+
+div.deleted {
+  background-color: #ffcccc;
+}
```
Comment (ericamador) [inline src/index.css]: Needs newline at end of file. Fix this in a rebase - don't add a new commit for small formatting issues like this.

Comment (kevinrobbins) [reply src/index.css]: I have already fixed this, I just haven't pushed it up yet.
